### ISSUE TITLE
More/Less Button text not change

### ISSUE THREAD

**Issue by SanatShukla** [loringdodge/ionic-item-expand#1]
I think you forgot to change method name from 
{{ isGroupShown(item) ? 'Less' : 'More' }} 
to 
{{ isItemShown(item) ? 'Less' : 'More' }}
Because 'isGroupShown' not available and not changing text More/Less.

**Comment by loringdodge** [comment]
You are absolutely right @SanatShukla . Thanks for pointing that out and the changes have been made. Thanks.
